plugin/mutate/source/dextool/plugin/mutate/backend/report/html/package.d

```diff
@@ -176,6 +180,21 @@ struct FileIndex {
         line.addClass("loc");
 
         line.addChild("span", "1:").addClass("line_nr");
+        auto mut_data = "var g_muts_data = {};\n";
+        mut_data ~= "g_muts_data[-1] = {'kind' : null, 'status' : null, 'testCases' : null, 'orgText' : null, 'mutText' : null, 'meta' : null};\n";
+        alias sort_tcs_on_kills = (a, b) => db.getTestCaseInfo(a, kinds)
```

Comment:
This leads to two database queries. A specialized query that returns the test cases sorted is more efficient.
I will make a note of it as a TODO to fix in the future.